src/qt/unlimiteddialog.cpp

```diff
@@ -192,9 +192,7 @@ void UnlimitedDialog::validateBlockSize()
     int mmb = ui.miningMaxBlock->text().toInt();
     int ebs = ui.excessiveBlockSize->text().toInt();
 
-    /* the mined block size must not be greater then the excessive block 
-     * size else the miner will be mining blocks she cannot track. */
-    if ( mmb > ebs )
+    if ( MiningAndExcessiveBlockValidatorRule(ebs, mmb))
```

Comment:
Suggest rename of this function to  `MiningAndExcessiveBlockValidateFail` or similar, so that it's clearer from the name that this is a bool which returns true if the validation fails.